### ISSUE TITLE
Adblock and screen rotation lock

### ISSUE THREAD

**Issue by guilhermsr-gr** [martykan/webTube#54]
I was looking for open source softwares, and the WT it's a great one, but I think can be better if added a adblock and a screen rotation lock as native.

**Comment by JasperWeiss** [comment]
Yeah, It'd be great if WT just skipped the ads. As in not loading the ad in the first place like NewPipe

**Comment by ImAnnoying** [comment]
Please make it optional - some of us want to support the people behind the videos we love

**Comment by mikeloeven** [comment]
Ad block is needed and in fact the main reason I looked into WebTube. The current adblocking youtube apps generally do not have a login function and lack ability to view or edit comments and keep track of subscribed channels like WebTube allows. i also agree with ImAnnoying that this should be a toggle able option
